packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.tsx

```diff
@@ -110,44 +110,127 @@ const ContractTypeGlossary = ({ category }: { category: string }) => {
+                                'If the duration is more than 24 hours, the Cut-off time and Expiry date will apply instead.'
+                            ),
+                        ],
+                    },
+                    { type: 'heading', text: localize('Cut-off time') },
+                    {
+                        type: 'paragraph',
+                        text: localize('Contracts will expire at exactly 14:00:00 GMT on your selected expiry date.'),
+                    },
+                    { type: 'heading', text: localize('Expiry date') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'Your contract will expire on this date (in GMT), based on the End time you’ve selected.'
+                        ),
+                    },
+                    { type: 'heading', text: localize('Contract value') },
+                    {
+                        type: 'paragraph',
+                        text: localize(
+                            'We’ll offer to buy your contract at this price should you choose to sell it before its expiry. This is based on several factors, such as the current spot price, duration, etc. However, we won’t offer a contract value if the remaining duration is below 24 hours.'
+                        ),
+                    },
+                ];
+                break;
             default:
                 content = [];
                 break;
         }
     }
     return (
         <React.Fragment>
-            {content?.map(({ type, text }: { type: string; text: string | string[] }) => {
-                if (type === 'heading' && typeof text === 'string') {
-                    return (
-                        <Text
-                            as='h2'
-                            key={text.substring(0, 10)}
-                            weight='bold'
-                            className='contract-type-info__content-glossary--heading'
-                        >
-                            {text}
-                        </Text>
-                    );
-                }
-                if (type === 'paragraph' && typeof text === 'string') {
-                    return (
-                        <Text as='p' key={text.substring(0, 10)}>
-                            {text}
-                        </Text>
-                    );
-                }
-                if (type === 'list' && typeof text !== 'string') {
-                    return (
-                        <ul key={text[0].substring(0, 15)}>
-                            {text.map(list_item_text => (
-                                <li key={list_item_text.substring(0, 20)}>{list_item_text}</li>
-                            ))}
-                        </ul>
-                    );
+            {content?.map(
+                ({ type, text }: { type: string; text: string | string[] | ComponentProps<typeof Localize> }) => {
```

Comment:
```suggestion
                ({ type, text }: { type: string; text: string | string[] | JSX.Element }) => {
```

Reply:
This won't work and break the rest of typescript in the file